api/metadata_endpoints.py

```diff
@@ -17,7 +18,11 @@
 from edr_pydantic.unit import Unit
 from edr_pydantic.variables import Variables
 from fastapi import HTTPException
-from grpc_getter import get_obs_request
+from grpc_getter import getTSAGRequest
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
Great to get logging in, but should we not grab the same log-level as set when running the server?

Reply:
Maybe I messed things up. Logging wasn't working at all for me, so I just added this stuff to get it working, and thought it might be useful, so I left it in.

What is the right way to do this? Where do I get it from the server? What server is that, uvicorn?

Reply:
It seems that the loglevel set when starting uvicorn is set globally. Gunicorn was not as easy. I think we should open a separate issue for implementing logging.

Reply:
So leave it like it is, or strip it out?

Reply:
I think you can just leave it for now.